.ci/jenkins_job_build.sh

```diff
@@ -50,8 +57,13 @@ mkdir -p $(dirname "${tests_repo_dir}")
 mkdir -p $(dirname "${kata_repo_dir}")
 [ -d "${kata_repo_dir}" ] || git clone "https://${kata_repo}.git" "${kata_repo_dir}"
 
-pushd "${kata_repo_dir}"
+# If CI running on bare mental, a few clean-up work before walking into test repo
+if [ "${BARE}" == true ]; then
+	clean_up_script="${tests_repo_dir}/.ci/${arch}/clean_up_${arch}.sh"
+	source "${clean_up_script}"
```

Comment:
Maybe add a check that the file exists before you try to source it, just in case we add an arch that is BARE but does not have a script in place. Just a safety measure.